.github/workflows/go-generate.yml

```diff
@@ -0,0 +1,21 @@
+on: [push, pull_request]
+name: go generate
+
+jobs:
+  gogenerate:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.16.x"
```

Comment:
A minor point: Would it make sense to use `1.15` since that's the version used by [`go.mod`](https://github.com/multiformats/go-multicodec/blob/master/go.mod#L3)?

Reply:
No, that's just the **minimum** version. Tools should be run with the most recent version.